### PR TITLE
DW-1148-cost-per-email-credits-plan

### PR DESCRIPTION
**Pull request by silsanchez8**
## Modifications: https://makingsense.atlassian.net/browse/DW-1148

* feat(PlanAgreement): Add cost per mail in prepaid plans
* Fix some Creditcard test wich was using an all function called _replaceAll_

image

### CODE REVIEW

File: src/components/ChangePlan/PlanCalculator/PlanAgreement/PlanAgreement.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const PlanAgreement = ({ planData }) => {
         </p>
       ) : null}
 
-      {planData.plan.type === 'monthly-deliveries' ? (
+      {planData.plan.type !== 'subscribers' ? (
```
Comment (cbernat) [inline src/components/ChangePlan/PlanCalculator/PlanAgreement/PlanAgreement.js:43]: So the value by email is mapped into the same field? I would prefer being explicit here, like this:
```suggestion
      {planData.plan.type === 'monthly-deliveries' && planData.plan.type === 'prepaid'  ? (
```
We have several plans, `subscribers`, `monthly-deliveries` `agencies`, `prepaid`, when I read planData.plan.type === 'monthly-deliveries' I have to think twice to understand that you are not including agencies or exactly to which plans you are pointing out.

**Comment by cbernat** [comment]
Would you mind adding a test to reinforce this mapping please? 😄

**Comment by silsanchez8** [comment]
> Would you mind adding a test to reinforce this mapping please? 😄
Ceci, there is already a test for theses cases: 
Do you think it's enough or maybe we have to create a new one?
Prepaid:
image

Subscribers:
image

Emails:

image

**Comment by cbernat** [comment]
🚀

**Pull request merged**